SU2_CFD/src/solvers/CTurbSASolver.cpp

```diff
@@ -583,8 +583,13 @@ void CTurbSASolver::BC_Inlet(CGeometry *geometry, CSolver **solver_container, CN
       conv_numerics->SetPrimitive(V_domain, V_inlet);
 
       /*--- Set the turbulent variable states (prescribed for an inflow) ---*/
+
+      //Solution_i[0] = nodes->GetSolution(iPoint,0);
+
```

Comment:
What happened here? bug?

Reply:
not used, it was used further down in the part "commented because of serious convergence problems".

Reply:
I see, we should delete all that commented out code at some point... it is unmaintainable by definition, please remove these new lines with comments then.